lib/ipmi_backend_utils.pm

```diff
@@ -30,7 +30,7 @@ our @EXPORT = qw(use_ssh_serial_console set_serial_console_on_xen switch_from_ss
 
 #use it after SUT boot finish, as it requires ssh connection to SUT to interact with SUT, including window and serial console
 sub use_ssh_serial_console {
-    console('sol')->disable;
+    console('sol')->disable if check_var('BACKEND', 'ipmi');
```

Comment:
I don't understand. This file is called "ipmi_backend_utils" but we call it from pvm which is not an ipmi-backend now? Looks strange

Reply:
It's because `use_ssh_serial_console` is used by spvm backend also. So what is better? Rename the library, put this function in another library or let it as is?